web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js

```diff
@@ -0,0 +1,213 @@
+            this.reload_current_view();
+            return result;
+        },
+        // defuse some functions we don't need
+        get_active_view: function()
+        {
+            return false;
+        },
+        load_views: function()
+        {
+            return jQuery.when();
+        },
+    });
+
+    widget_one2many_tag.web.form.widgets.add(
+        'one2many_tags',
+        'widget_one2many_tag.web_widget_one2many_tags.FieldOne2ManyTags'
+    );
+
+    widget_one2many_tag.web.list.One2ManyTags = widget_one2many_tag.web.list.Column.extend({
+        _format: function (row_data, options)
+        {
+            if(!_.isEmpty(row_data[this.id].value) && row_data[this.id + '__display'])
+            {
+                row_data[this.id] = row_data[this.id + '__display'];
+            }
+            return this._super(row_data, options);
+        },
+    });
+
+    widget_one2many_tag.web.list.columns.add(
+        'field.one2many_tags',
+        'widget_one2many_tag.web.list.One2ManyTags'
+    );
+
+    widget_one2many_tag.web.ListView.List.include({
+        render_cell: function (record, column)
+        {
+            if(column.widget == 'one2many_tags')
+            {
+                var dataset = new widget_one2many_tag.web.form.One2ManyDataSet(
+                        this, column.relation),
+                    fake_widget = {
+                        dataset: dataset,
+                        trigger_on_change: function() {},
+                        get_active_view: function () {},
+                        _super: function() {},
+                        set: function() {},
+                        build_context: function()
+                        {
+                            return new widget_one2many_tag.web.CompoundContext(
+                                column.context
+                            );
+                        },
+                    },
+                    value = record.get(column.id);
+                dataset.o2m = fake_widget;
+                odoo.web_widget_one2many_tags.FieldOne2ManyTags
+                    .prototype.set_value.apply(fake_widget, [value])
+                dataset.read_ids(dataset.ids, ['display_name'])
+                .then(function(names) {
+                    if(!names.length)
+                    {
+                        return;
+                    }
+                    record.set(
+                        column.id + '__display',
+                        _(names)
+                            .map(function(name)
+                            {
+                                return name.display_name ||
+                                    widget_one2many_tag.web._t('New record');
+                            })
+                            .join(', ')
+                    );
+                });
+                column = _(column).extend({type: 'one2many_tags'});
+            }
+            return this._super(record, column);
+        },
+    });
```

Comment:
you don't return anything, this means the classes defined here are not accessible to other addons